### ISSUE TITLE
[Developer] kmcomp fails to merge .keyboard_info if a package doesn't contain .kmx file

### ISSUE THREAD

**Issue by darcywong00** [keymanapp/keyman#1110]
Investigating why the CI fails to build keymanapp/keyboards/pull/381

The keyboard is mobile only, so the .kmp should only have .js files and no .kmx files.

This causes MergeKeyboardInfo.pas to fail at lines 802 with the error
```
Memory 0: 00000000 Invalid .keyboard_info file: There should be exactly 
1 .kmx file in the package.
```

Can be repro'd with a minimal keyboard

### moo.kmn
```
store(&NAME) 'moo'
store(&TARGETS) 'mobile tablet'
store(&COPYRIGHT) '2018'
store(&LAYOUTFILE) 'moo.keyman-touch-layout'
begin Unicode > use(main)

group(main) using keys
```

... create a moo.kps package that contains the .js file ...

### moo.keyboard_info
```
{
    "id" : "moo",
    "name" : "moo",
    "license": "mit",
    "languages": ["ood-Latn"],
    "description": "This is a sample description."
}
```

**Comment by darcywong00** [comment]
I suspect we haven't encountered this before because:
1. Many of the web-only keyboards in the keyboards repo don't generate packages (see release/fv)
2. All the keyboard packages have `any` or `windows` as a keyboard target, so they end up having a .kmx file.

**Comment by darcywong00** [comment]
Fixed in 10.0 stable by #1174 
Fixed in 11.0 alpha by #1158
